croptalk/load_data.py

```diff
@@ -3,6 +3,7 @@
 import pandas as pd
 from sqlalchemy import create_engine, text
 from dotenv import load_dotenv
+from croptalk.utils import query_sql_data
 
 load_dotenv("secrets/.env.secret")
```

Comment:
I don't think this is now used anywhere... should we remove it?

Same for all the imports in this module, except query_sql_data